modules/microshift-accessing-cluster-remotely-non-admin.adoc

```diff
@@ -25,6 +23,17 @@ Use the following procedure to access the {product-title} cluster from a remote
 [user@microshift]$ sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp && sudo firewall-cmd --reload
 ----
 
+.Verify {product-title} is running
+
+. As `user@microshift`, verify that {product-title} is running by running the following command:
++
+[source,terminal]
+----
+[user@workstation]$ oc get all -A
+----
+
+.Procedure
```

Comment:
Do we need to split procedures? The above and below steps must be taken one after another, so arguably it's the same procedure.

Reply:
From a mod docs perspective, yes we do.